exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,20 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0){
+    return true;
+  }
+  else return false;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if (number >= 0 && number <= 10){
+    return true;
+  }
+  else return false;
+}
 
```

Comment:
plz. think about how you can make these functions shorter, hint: think about what the value and the type of this expression: number < 0